### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them,  @I534472

File: src/xcode/ENA/ENA/Resources/Localization/ro.lproj/Localizable.strings [hunk 1]
```diff
@@ -655,30 +655,20 @@
...
-
 "Home_Risk_Inactive_NoCalculation_Possible_Button" = "Activați înregistrarea în jurnal a expunerilor";
 
 "Home_Risk_Inactive_Outdated_Results_Button" = "Activați înregistrarea în jurnal a expunerilor";
 
 /* Home Low card */
 "Home_Risk_Low_Title" = "Risc redus";
 
-"Home_Risk_Low_SaveDays_Item_Title" = "În ultimele %1$u zile, înregistrarea în jurnal a expunerilor a fost activă timp de %2$u zile.";
+"Home_Risk_Low_SaveDays_Item_Title" = "În ultimele 14 zile, înregistrarea în jurnal a expunerilor a fost activă timp de %1$u zile.";
```
Comment (PeterBuchholz) [inline src/xcode/ENA/ENA/Resources/Localization/ro.lproj/Localizable.strings:665]: This as already corrected in 1.2.0 in stead of the 14 there should be the placeholder %2$u
See: https://github.com/corona-warn-app/cwa-app-ios/pull/1018/files

**Review by PeterBuchholz — COMMENTED** [review]
romanian language is missing a placeholder again

File: src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings [hunk 2]
```diff
@@ -961,7 +961,7 @@
 "ENATanInput_CharacterIndex" = "Символ %1$u от %2$u";
 
 /* Active Tracing Interval Formatting:\n The following two entries are used together. If tracing was not active (almost) 14 days\n during the last 14 days then `Active_Tracing_Interval_Partially_Active` is used – otherwise\n `Active_Tracing_Interval_Partially_Active` will be used. */
-"Active_Tracing_Interval_Partially_Active" = "Регистрирането на излагания на риск беше активно през %1$u от изминалите %2$u дни.";
+"Active_Tracing_Interval_Partially_Active" = "Регистрирането на излагания на риск е било активно през %1$u от изминалите 14 дни.";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings:964]: this change is wrong

File: src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings [hunk 3]
```diff
@@ -662,7 +662,7 @@
 /* Home Low card */
 "Home_Risk_Low_Title" = "Нисък риск";
 
-"Home_Risk_Low_SaveDays_Item_Title" = "Регистрирането на излагания на риск беше активно през %1$u от изминалите %2$u дни.";
+"Home_Risk_Low_SaveDays_Item_Title" = "Регистрирането на излагания на риск е било активно през %1$u от изминалите 14 дни.";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings:665]: This change is wrong.

**Review by steins-code — CHANGES_REQUESTED** [review]
please update the commit

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them,  @I534472

**Comment by steins-code** [comment]
> This pull request is for translation delivery. Please review and approve them, @I534472

I resolved the conflicts in another PR #1050  since I cannot commit to this PR

**Pull request closed without merging**